products/sle/main.pm

```diff
@@ -1215,8 +1204,20 @@ elsif (get_var("Y2UITEST")) {
     # setup $serialdev permission and so on
```

Comment:
@coolo you right , I wanted to say that I fail to find test suite which using this variable , so I suspecting that there is no job which run tests which are in this if branch. Sorry for confusion.